src/OrchardCore/OrchardCore.Email.Abstractions/MailMessageAttachment.cs

```diff
@@ -16,5 +17,10 @@ public class MailMessageAttachment
         /// Gets or sets the attachment file stream.
         /// </summary>
         public Stream Stream { get; set; }
```

Comment:
I think the issue is that there is a Stream property instead of just an `OpenStream` method. If there was then the caller would be responsible for closing it instead.